codec-http/src/main/java/io/netty/handler/codec/http/cookie/ServerCookieEncoder.java

```diff
@@ -141,19 +165,19 @@ public String encode(Cookie cookie) {
      * @param cookies a bunch of cookies
      * @return the corresponding bunch of Set-Cookie headers
      */
-    public List<String> encode(Collection<? extends Cookie> cookies) {
-        if (checkNotNull(cookies, "cookies").isEmpty()) {
+    public List<String> encode(Cookie... cookies) {
```

Comment:
Could we add a new method with a param to disable ordering and so remove the overhead for people who not care?